### PR TITLE
PEP8: Fixing best.py file

### PR DESCRIPTION
**Pull request by springcoil**
I fixed up the best.py file. I'll work on some others.

### CODE REVIEW

**Comment by springcoil** [comment]
Need to mention this https://github.com/pymc-devs/pymc3/issues/1318

File: pymc3/examples/best.py [hunk 1]
```diff
@@ -2,62 +2,81 @@
...
-    diff_of_stds = pm.Deterministic('difference of stds', group1_std - group2_std)
-    effect_size = pm.Deterministic('effect size', diff_of_means / pm.sqrt((group1_std**2 + group2_std**2) / 2))
+    group2 = pm.StudentT('placebo', nu=nu, mu=group2_mean, lam=lam2,
+                         observed=y2)
+
+    diff_of_means = pm.Deterministic('difference of means', group1_mean -
+                                     group2_mean)
+    diff_of_stds = pm.Deterministic('difference of stds',
+                                    group1_std - group2_std)
+    effect_size = pm.Deterministic('effect size',
```
Comment (twiecki) [inline pymc3/examples/best.py]: Those are bit odd, how about:

``` python
effect_size = pm.Deterministic(
    'effect size', 
    diff_of_means / pm.sqrt((group1_std**2 + group2_std**2) / 2)
)
```

**Comment by springcoil** [comment]
@twiecki I fixed it - thanks for the feedback :)

**Comment by twiecki** [comment]
:+1:

**Pull request merged**